### DISCUSSION TITLE
Can't activate Copilot

### DISCUSSION THREAD

**Discussion by ghost** [community/community#10671] [Copilot Conversations]
Tell me please. I have been trying to connect Copilot for 5 months now. It says "Join the GitHub Copilot waitlist"... What can I do to start using Copilot????

**Comment by vruzin** [comment]
I have 2 accounts, so I also write from this account. I really want to try Copilot!!!!

**Comment by caioagiani** [comment]
You need to register (https://copilot.github.com, https://github.com/features/copilot/signup) and wait for approval, maybe try to resubmit your application

![image](https://user-images.githubusercontent.com/56451511/151487999-5cde2fb2-46f1-4e06-a276-1f3a8b271853.png)

**Comment by imdevarsh** [comment]
Have you signed up through https://github.com/features/copilot/signup yet? That's how I signed up to the waitlist. You then will need to wait for some time until your account gets in.
